client/package.json

```diff
@@ -88,13 +92,16 @@
     "eslint-plugin-prettier": "^3.1.3",
     "eslint-plugin-react": "^7.20.0",
     "eslint-plugin-react-hooks": "^4.0.4",
+    "expect-puppeteer": "^4.4.0",
```

Comment:
added these 3 libraries for better testing developer experience, and allow us to use `expect-puppeteer`'s API instead of puppeteer's error prone low level API:

https://github.com/smooth-code/jest-puppeteer/tree/master/packages/expect-puppeteer